views/components/tabular-data.html

```diff
@@ -0,0 +1,29 @@
+{% macro tabularData(myLatest) %}
+<article class="base--article" itemscope itemtype="http://schema.org/Table">
+  <header class="base--header">
+    <h1 class="base--h1" itemprop="name">{{myLatest.title}}</h1>
```

Comment:
Why does the table have both a header with a `h1` _and_ a `caption`? A `caption` is the title of a table.